hpy/tools/autogen/public_api.h

```diff
@@ -295,6 +297,39 @@ int HPyTracker_Add(HPyContext *ctx, HPyTracker ht, HPy h);
 void HPyTracker_ForgetAll(HPyContext *ctx, HPyTracker ht);
 void HPyTracker_Close(HPyContext *ctx, HPyTracker ht);
 
+/* HPyField */
+
+/* TODO / Open questions:
+
+  - what is the semantics w.r.t. overwriting HPyFields? I.e.:
+
+        pair->a = HPyField_Store(ctx, h1);
```

Comment:
If  we used `HPyField_Store(ctx, h1, &pair->a)` then `HPyField_Store` can call clear on whatever was in `a` before overwriting it? Or perhaps raise an error if it was not cleared? That definitely seems like a good feature for debug mode if no where else.

Reply:
this comment no longer applies to the current implementation